static/js/hotkey.js

```diff
@@ -126,7 +126,7 @@ exports.get_keydown_hotkey = function (e) {
     }
 
     var isCmdOrCtrl = /Mac/i.test(navigator.userAgent) ? e.metaKey : e.ctrlKey;
-    if (isCmdOrCtrl) {
+    if (isCmdOrCtrl && !e.shiftKey) {
```

Comment:
I think we might want something similar for `keydown_ctrl_mappings` above as well?

Reply:
@timabbott OK, I've added the same logic to the `keydown_ctrl_mappings` handler, and added a new test. Please let me know if there is more that I should change or fix. Thanks!